projects/packages/jetpack-mu-wpcom/src/class-jetpack-mu-wpcom.php

```diff
@@ -262,9 +262,13 @@ public static function load_coming_soon() {
 		 * Explicitly pass $markup = false in get_plugin_data to avoid indirectly calling wptexturize that could cause unintended side effects.
 		 * See: https://developer.wordpress.org/reference/functions/get_plugin_data/
 		 */
+		$fse_plugin                 = 'full-site-editing/full-site-editing-plugin.php';
+		$fse_plugin_path            = WP_PLUGIN_DIR . '/' . $fse_plugin;
 		$invalid_fse_version_active =
-			is_plugin_active( 'full-site-editing/full-site-editing-plugin.php' ) &&
-			version_compare( get_plugin_data( WP_PLUGIN_DIR . '/full-site-editing/full-site-editing-plugin.php', false )['Version'], '3.56084', '<' );
+			is_plugin_active( $fse_plugin ) &&
+			file_exists( $fse_plugin_path ) &&
```

Comment:
looks like we have to check it before `is_plugin_active`?

Reply:
IIRC is_plugin_active() checks site options, but moving it in front doesn't hurt 🙂
The issue is that `full-site-editing/full-site-editing-plugin.php` is still in site options but the plugin file doesn't exist anymore.